modules/messages/client/components/Inbox.component.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import * as api from '../api/messages.api';
+import Avatar from '@/modules/users/client/components/Avatar.component';
+import Activate from '@/modules/users/client/components/Activate';
+
+const userType = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  username: PropTypes.string.isRequired,
+  displayName: PropTypes.string.isRequired,
+});
```

Comment:
We could start to put these propTypes into places where they can be reused across the app... maybe in 'modules/users/client/components/users.prop-types.js`?

Reply:
Valid! Could also be ok follow up PR.

Reply:
I did it already, as I needed it to be in a seperate file when I extracted the `InboxThread` component. I put that one in `/modules/users/client/users.prop-types` (as it's not a component... ). Will do for now at least.